src/components/blocks/DlLabelPicker/DlLabelPicker.vue

```diff
@@ -125,9 +129,8 @@ export default defineComponent({
 
         const inputValue = ref('')
         const currentSelectedLabel = ref<DlLabelPickerItem>(
-            items.value ? items.value[0] : null
+            (defaultSelectedLabel.value ?? items.value?.[0]) || null
```

Comment:
Good catch @guyDataloop it should be default of Null instead of undefined as to avoid issues.

defaultSelected in this case @guyDataloop is not the undefined prop itself but a reference to it so there is no need for a ? after defaultselected to access its value